docs/pages/deploy-a-cluster/multi-region-blueprint.mdx

```diff
@@ -0,0 +1,314 @@
+---
+title: "Multi-region blueprint"
+description: "Blueprint describing how to deploy a multi-region Teleport Enterprise cluster using CockroachDB."
+---
+
+## Introduction
+
+This page describes how to deploy a Teleport cluster in multiple regions to
+improve resiliency and sustain regional failure. This is not a step-by-step
+guide but a blueprint on how to build a multi-region Teleport cluster. This
+blueprint must be adapted to work with your specific infrastructure and network
+constraints.
+
+## Important technical consideration
+
+Before continuing, you must acknowledge the following warnings:
+
+- This setup is not required for most Teleport users and customers.
+  Single-region multi-zone deployments provide the best availability/cost ratio.
+  For example, Amazon SLAs are 99.99% for both DynamoDB and EC2.
+- This blueprint relies on many complex components. If you don't have the team
+  capacity and expertise to deploy and operate such components, you will end
+  up with a lower availability than a single-region multi-zone setup.
+- You are responsible for deploying and maintaining the Teleport dependencies
+  including the database, object storage, container runtime, network plugins, ...
+
+<Admonition type="tip">
+
+For the teams with limited bandwidth who need multi-region clusters, Teleport
+offers to host cross-region clusters. You can benefit from the
+setup described in this document without the operational costs and complexity,
+see [the Teleport Enterprise Cloud page](../choose-an-edition/teleport-cloud/introduction.mdx)
+for more details.
+
+</Admonition>
+
+## Architecture overview
+
+A multi-region Teleport deployment is composed of several regional Teleport
+deployments. Teleport stores its state in a multi-region backend and uses
+[Proxy Peering](../architecture/proxy-peering.mdx) to connect users to resources
+hosted a different region.
+
+The recommended multi-region backend
+is [the CockroachDB backend](../reference/backends.mdx#cockroachdb)
+available in [Teleport Enterprise](../choose-an-edition/teleport-enterprise/introduction.mdx).
+
+![Architecture diagram]()
+
+To run a multi-region Teleport deployment, you must have:
+
+- **3 regions**. Teleport only needs 2, but CockroachDB needs 3 for its quorum mechanism.
+- Teleport Proxy Service instances able to dial each other by IP address.
+  This means you have **cross-region Pod/Instance connectivity**. This is typically
+  achieved with VPC peering and/or service mesh.
+- A **multi-region object storage** for session recordings.
+- **GeoDNS** to route users and Teleport agents to the closest Teleport cluster.
+- A **multi-region CockroachDB cluster**. You can use the CockroachDB core
+  features, but optimal performance and fault-tolerance is achieved by using
+  [the multi-region features from CockroachDB Enterprise](https://www.cockroachlabs.com/docs/stable/enterprise-licensing).
```

Comment:
I think that as we say CockroachDB core is not recommended for multi-region production use below, we should be explicit about that here too.